cmd/dlv/dlv_test.go

```diff
@@ -1071,6 +1072,11 @@ func TestTracePid(t *testing.T) {
 	if !bytes.Contains(output, expected) {
 		t.Fatalf("expected:\n%s\ngot:\n%s", string(expected), string(output))
 	}
+
+	err = targetCmd.Process.Signal(syscall.Signal(0))
```

Comment:
This doesn't work on windows. I'd call targetCmd.Process.Wait on a separate goroutine and close a channel when it returns, here I'd check that the channel doesn't get closed for, say 500ms (a select between reading on the channel and reading on time.After).

Doing the check immediately races with the operating system: the process struct could still be around for a while after the process has died and if we wait a while and do FindProcess again the pid could have been reused for something else.

Reply:
AFAICT, the only CI failure is related to the linux/loong64 builder which we are trying to add (so far unsuccessfully).